services/121-service/test/performance/Import1000Registrations.test.ts

```diff
@@ -0,0 +1,31 @@
+import { HttpStatus } from '@nestjs/common/enums/http-status.enum';
+
+import { SeedScript } from '@121-service/src/scripts/enum/seed-script.enum';
+import { importRegistrationsCSV } from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+
+const csvFilePath =
+  './test-registration-data/test-registrations-westeros-1000.csv';
+
+jest.setTimeout(450000); // 7,5 minutes
```

Comment:
```suggestion
jest.setTimeout(450_000); // 7.5 minutes
```

Hope this underscoring works (it is better readable)